test/integration.test.js

```diff
@@ -417,3 +429,19 @@ const debugDumpData = (name, data) => {
   for (const k of keys) sorted[k] = data[k]
   debug(name, JSON.stringify(sorted, null, 2))
 }
+
+const fetchMeasurementsWithCache = async (cid) => {
```

Comment:
I copied this function from [bin/dry-run.js](https://github.com/filecoin-station/spark-evaluate/blob/53c7923ded7d398266b97491c583e3da5d2621f8/bin/dry-run.js).

If you think it's worth extracting this function into a shared helper, then let me know, and I'll open a follow-up pull request.

Reply:
Fine with me as is 👍 It's small in scope and not complex